docs/en-us/workloadspread.md

```diff
@@ -3,79 +3,112 @@ title: WorkloadSpread
 ---
 # WorkloadSpread
 
-WorkloadSpread can be used to constrain the spread of stateless workload, which empower single workload the abilities for
-multi-domain deploy and elastic deploy.
+WorkloadSpread can distribute Pods of workload to different types of Node according to some rules, which empowers single workload the abilities for
+multi-domain deployment and elastic deployment.
 
-The feature of WorkloadSpread is similar with UnitedDeployment in Kruise community. Each WorkloadSpread defines multi-domain
+Some common rules include:
+- Horizontal spread (for example, average spread in dimensions such as host, az, etc.)
+- Spread according to the specified ratio (for example, deploy Pod to several specified az according to the proportion)
+- Different subset management with priority, such as
+  - deploy Pods to ecs first, and deploy to eci when its resources are insufficient.
+  - deploy a fixed number of Pods to ecs first, and the rest Pods are deployed to eci.
+
+The feature of WorkloadSpread is similar with UnitedDeployment in OpenKruise community. Each WorkloadSpread defines multi-domain
```

Comment:
> Each domain should at least provide the capacity to run the replicas number of pods called `maxReplicas`.
should or may ? if maxReplicas is not provided, workloadspread can still spread the pods according to the capacity of the subset.